.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Set up go
         uses: actions/setup-go@v3
         with:
-          go-version: '1.18'
+          go-version: '1.19'
```

Comment:
Do we have a reason why we don't want to migrate to `go-version-file: 'go.mod'`?

Reply:
No! Let's do it!

Reply:
Updated in 7cf318e3